$NetBSD: patch-ab,v 1.6 2001/03/19 22:11:15 wennmach Exp $

Get rid of ugly "deprecated symbol unix no longer predefined" warnings.

--- syscfg.h.orig	Mon Mar 19 15:01:36 2001
+++ syscfg.h	Mon Mar 19 15:05:43 2001
@@ -197,7 +197,11 @@
 
 /* Note: may not catch all IBM AIX compilers or SCO compilers */
 /* cc on NetBSD doesn't define unix anymore :-/ */
-#if defined(unix)|| defined(__unix) || defined(_AIX) || defined(__NetBSD__) || defined(SCO)
+#if defined(__NetBSD__)
+#undef unix
+#define unix
+#endif
+#if defined(unix)|| defined(__unix) || defined(_AIX) || defined(SCO)
 # ifndef unix
 #  define unix
 # endif
